src/smefit/optimize/ultranest.py

```diff
@@ -71,12 +71,7 @@ def __init__(
         single_parameter_fits,
         pairwise_fits,
         use_multiplicative_prescription,
-        live_points=500,
-        lepsilon=0.001,
-        target_evidence_unc=0.5,
-        target_post_unc=0.5,
-        frac_remain=0.01,
-        store_raw=False,
+        ns_settings
```

Comment:
A bit more picky comment, the whole point of having the `from_dict` method was for to group variables in the runcards, but leaving them spelled out in the class constructor to make more clear, which are the relevant one. 
I'd prefer to keep the default values here.